src/meta/client/MetaClient.cpp

```diff
@@ -719,7 +720,8 @@ MetaClient::createEdgeSchema(GraphSpaceID spaceId, std::string name, nebula::cpp
 }
 
 folly::Future<StatusOr<bool>>
-MetaClient::alterEdge(GraphSpaceID spaceId, std::string name,
+MetaClient::alterEdgeSchema(GraphSpaceID spaceId,
+                      std::string name,
```

Comment:
alignment

Reply:
Good catch， 👍
ps : Last night I found a flaw in the original code, I submitted it together.